### PR TITLE
Updates all dependencies

### PR DESCRIPTION
**Pull request by dimitropoulos**
This one was tricky due to some breaking changes in connect-es.  While I'm not terribly happy with the solution, it appears to work, the required changes for the test `useMutation -> makes a mutation` in `unary-hooks.ts` bothers me a lot.

### CODE REVIEW

File: packages/connect-query/jest.config.ts [hunk 1]
```diff
@@ -16,7 +16,8 @@ import type { Config } from 'jest';
 
 const config: Config = {
   preset: '../../jest-preset.js',
-  testEnvironment: 'jsdom',
+  // testEnvironment: 'jsdom',
```
Comment (paul-sachs) [inline packages/connect-query/jest.config.ts:19]: Maybe delete comment... Or keep it, up to you.

File: packages/connect-query/src/unary-hooks.test.ts [hunk 2]
```diff
@@ -878,8 +878,8 @@ describe('unaryHooks', () => {
         expect(result.current.get.isSuccess).toBeTruthy();
       });
 
-      expect(result.current.get.data?.count).toStrictEqual(1n);
       expect(result.current.mut.data).toStrictEqual(undefined);
+      expect(result.current.get.data?.count).toStrictEqual(0n);
```
Comment (paul-sachs) [inline packages/connect-query/src/unary-hooks.test.ts:882]: Did these tests need to change? Not sure i understand why they would...

**Review by paul-sachs — APPROVED** [review]
Very minor questions, nothing blocking.

File: packages/connect-query/src/unary-hooks.test.ts (hunk 2 above)
Comment (dimitropoulos) [reply packages/connect-query/src/unary-hooks.test.ts:882]: SAME.  that's why I called it out.  This is what it takes to make them pass and I'm pretty confused as to why.

File: packages/connect-query/jest.config.ts (hunk 1 above)
Comment (dimitropoulos) [reply packages/connect-query/jest.config.ts]: good call, I missed this.

File: packages/connect-query/src/unary-hooks.test.ts (hunk 2 above)
Comment (dimitropoulos) [reply packages/connect-query/src/unary-hooks.test.ts:882]: update: paul and I talked and decided to move forward.  we have other things than this to investigate going forward next week, anyway.

**Pull request merged**